### DISCUSSION TITLE
Auto add to album

### DISCUSSION THREAD

**Discussion by jfreak53** [immich-app/immich#9836] [Q&A] [answered]
Is there a way for a users pictures to automatically get added to X album on upload? Just that one user, to just a specific album.

**Comment by bo0tzz** [comment, marked as answer]
Not yet, this will be possible once smart rules are implemented.

**Reply by jfreak53** [reply]
Any idea on timeframe for that?

**Reply by bo0tzz** [reply]
Nope, we don't do timeframes ;) The work has been started, but is on hold for a bit as we sidequest into tidying up the permission system etc.

**Comment by NicholasFlamy** [comment]
#1673
